src/stickybits.js

```diff
@@ -199,8 +199,13 @@ Stickybits.prototype.computeScrollOffsets = function computeScrollOffsets(item)
     stickyStart = parent.getBoundingClientRect().top - scrollElOffset
   }
   it.offset = scrollElOffset + p.stickyBitStickyOffset
+  let stickyStop = (stickyStart + parent.offsetHeight) - (it.el.offsetHeight - it.offset)
+  if ( iw ) {
+	  stickyStart = stickyStart - it.offset
+	  stickyStop = stickyStart + parent.offsetHeight - it.el.offsetHeight
```

Comment:
Is it only the `it.offset` that is creating a problem for `stickyStart`? 

Why is `stickyStop` adjusted?